examples/seismic/tti/operators.py

```diff
@@ -277,31 +275,31 @@ def kernel_staggered_2d(model, u, v, **kwargs):
 
     if forward:
         # Stencils
-        phdx = costheta * u.dx - sintheta * u.dy
```

Comment:
why do we need the extra 'c' now? 

is this affecting use side anyhow? looks like a critical change at first glance?

Reply:
If you check the 3d below you'll see the `c` was supposed to be there but was luckily working in 2D without it. t's mostly because this is a twisted PDE with a missing field so need some tweaks for the FD>